packages/jsii/lib/assembler.ts

```diff
@@ -226,7 +226,12 @@ export class Assembler implements Emitter {
     }
 
     async function _loadReadme(this: Assembler) {
-      const readmePath = path.join(this.projectInfo.projectRoot, 'README.md');
+      // Search for `README.md` in a case-insensitive way
+      const fileName =
+        (await fs.readdir(this.projectInfo.projectRoot)).find(
+          (file) => file.toLocaleLowerCase() === 'readme.md',
+        ) ?? 'README.md';
+      const readmePath = path.join(this.projectInfo.projectRoot, fileName);
```

Comment:
It feels a bit weird you read the files in the directory, then check to see if the path exists. I think the below reads a bit cleaner. Also, why the `await fs.readdir` vs `fs.readdirSync`?

```suggestion
      // Search for `README.md` in a case-insensitive way
      const fileName =
        fs.readdirSync(this.projectInfo.projectRoot).find(
          (file) => file.toLocaleLowerCase() === 'readme.md',
        );
      if (!fileName) { return undefined; }
      const readmePath = path.join(this.projectInfo.projectRoot, fileName);
      // Skip the fs.pathExists... check below
```

Reply:
You're right. I guess I've been lazy 👍